src/components/20-molecules/datepicker/index.js

```diff
@@ -266,7 +267,20 @@ class AXADatepicker extends NoShadowDOM {
 
     this.setMonthAndYearItems(month, year);
 
-    const { width = 'auto', height = '40px', error, invalid } = this;
+    const { width = 'auto', height = '40', error, invalid } = this;
+
+    // % on html width attribute not allowed
+    if (width.indexOf('%') > -1) { // TODO: setStyle
```

Comment:
`width.includes()` would be nicer and more readable

Reply:
include is not supported by IE

Reply:
Thats polyfilled. But i will not use it because it is a array function. It only works because strings are internally arrays of chars. @LucaMele what do you mean?

Reply:
`width.indexOf('%') > -1` is much better as includes is for arrays